pkgs/development/compilers/jetbrains-jdk/jcef.nix

```
@@ -76,8 +76,8 @@ buildType = if debugBuild then "Debug" else "Release";
 
 in stdenv.mkDerivation rec {
   name = "jcef-jetbrains";
-  rev = "153d40c761a25a745d7ebf0ee3a024bbc2c840b5";
-  commit-num = "611";  # Run `git rev-list --count HEAD`
+  rev = "3dfde2a70f1f914c6a84ba967123a0e38f51053f";
+  commit-num = "654";  # Run `git rev-list --count HEAD`
```

```suggestion
  version = "654";  # Run `git rev-list --count HEAD`
```
How about we use the commit number as the version? Before the package did not have one at all. This also solves the question how to write the commit message